glossary.md

```diff
@@ -189,6 +189,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | stochastic gradient descent  | hạ gradient ngẫu nhiên                                         |                       |
 | speech recognition           | nhận dạng giọng nơi                                            |                       |
 | supervised learning          | học có giám sát                                                |                       |
+| surrogate objective          | mục tiêu đại diện                                              | http://bit.ly/2PIxkN1                      |
```

Comment:
@ngcthuong @william-vu @nmdang vào giúp dịch cụm này.

Mọi người chắc đều biết từ này dùng ở đâu, mình vẫn xin nhắc lại cho những ai chưa biết.

Xét ví dụ với bài toán phân loại. Với con người thì đơn giản là lấy độ chính xác làm thước đo để tối ưu. Tuy nhiên với máy thì hàm tính độ chính xác là một hàm rời rạc nên rất khó tối ưu. Bởi vậy, người ta phải tìm một hàm thay thế cho dễ tối ưu. Hàm thay thế thường liên tục và dễ tính gradient. Hàm này không phải là hàm đại diện cho hàm ban đầu mà chỉ là một hàm thay thế. 

Đọc thêm https://www.quora.com/Whats-a-surrogate-in-machine-learning.

Một phần khác (#141) đang bị block bởi cụm này nên mình tập trung xong PR này trước.

Reply:
Em đọc kiểu gì cũng thấy từ "đại diện" giống với cái anh đang mô tả. Hình như em với anh đang hiểu nghĩa từ `đại diện` ngược với nhau. :))
Theo em hiểu thì cái "đại diện" sẽ được sử dụng như cái gốc khi cái gốc không khả dụng.
ví dụ: luật sư đại diện cho thân chủ trước toà (đễ bào chữa vì thân chủ không giỏi cãi nhau), giáo hoàng đại diện cho chúa trước tín đồ (để ban phước vì chúa ... ít khi xuất hiện), v.v
-> hàm entropy chéo đại diện cho hàm accuracy (để tối ưu vì hàm accuracy không liên tục)

Reply:
@tiepvupsu :nếu buộc phải dịch thì như @lkhphuc  gợi ý dùng từ đại diện; một từ khác có thể dùng, ấy là từ "thay thế". Từ "thay thế" có bao hàm là cái gốc không available/không dùng được nên mới phải dùng cái khác thay nó & diễn tả được các context mà từ "surrogate" hay xuất hiện. Từ "đại diện" thì khi đọc nên e dễ nghĩ đến từ "representative" hơn.

Reply:
Em quên không nói trong comment trước là em chọn dịch là "thay thế". Em cũng hiểu hai từ "thay thế" và "đại diện" như bác nói.

Reply:
Từ "thay thế" em thấy vẫn đúng, chỉ là nó không sát nghĩa hơn từ "đại diện" theo cách em hiểu thôi.
Nếu không ai có thêm ý tưởng gì mới thì mình bây giờ cứ dịch là "thay thế" cũng được. 
```suggestion
| surrogate objective          | mục tiêu thay thế                                              | http://bit.ly/2PIxkN1                      |
```